pub/lifecycle_test.go

```diff
@@ -13,11 +13,11 @@ func TestAuthorizeCaller(t *testing.T) {
 
 	httpmock.RegisterResponder("GET", "http://localhost:7202/lifecycle/api/v1/auth/can-call-job/skynet/latest",
 		httpmock.NewStringResponder(200,
-			`{"id": "000", "name": "skynet", "version": "0.0.1", "status": "running", "create_time": 1000, "update_time": 1000, "manifest": null, "internal_name": "adder-v-0-0-1"}`))
+			`{"job": {"id": "000", "name": "skynet", "version": "0.0.1", "status": "running", "create_time": 1000, "update_time": 1000, "manifest": null, "internal_name": "adder-v-0-0-1"}, "caller": "bob"}`))
 
 	httpmock.RegisterResponder("GET", "http://localhost:7202/lifecycle/api/v1/auth/can-call-job/skynet/0.0.1/api/v1/perform",
 		httpmock.NewStringResponder(200,
-			`{"id": "000", "name": "skynet", "version": "0.0.1", "status": "running", "create_time": 1000, "update_time": 1000, "manifest": null, "internal_name": "adder-v-0-0-1"}`))
+			`{"job": {"id": "000", "name": "skynet", "version": "0.0.1", "status": "running", "create_time": 1000, "update_time": 1000, "manifest": null, "internal_name": "adder-v-0-0-1"}, "caller": "bob"}`))
```

Comment:
Do you have here test for empty caller, ie for un-authorized calls? Would be useful to see that nothing breaks also in this case.